src/server/pfs/server/cache_stats.go

```diff
@@ -0,0 +1,87 @@
+package server
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"sync"
+
+	"github.com/fatih/camelcase"
+	"github.com/golang/groupcache"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
+)
+
+type cacheStats struct {
+	cacheName    string
+	descriptions map[string]*prometheus.Desc
+	*groupcache.Stats
```

Comment:
Any particular reason you choose to embed this rather than just having it be a field? I don't think it really affects anything but I normally only embed for a particular reason. I also think you could drop the `*` here and things would all work the same. Both minor points, neither is really necessary before merging.